### DISCUSSION TITLE
Colour Todays Date

### DISCUSSION THREAD

**Discussion by mikekenyon99** [totaldebug/atomic-calendar-revive#1450] [Showroom]
Hi - is it possible to colour the current date?  Ideally I want to change the colour of todays date (14th) to blue and the rest of the dates (15,16,17th) to black.  Thanks

Mike

**Comment by realtek2017** [comment]
Yes, you need to install card_mod and then use:

```
card_mod:
  style: |
    .current-day  { 
    color: orange !important;
    }
```
